<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDimensionParameters</class>
 <widget class="QWidget" name="CDimensionParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CDimensionParameters</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../DyssolMainWindow/Resources.qrc">
    <normaloff>:/Icons/WinIcon</normaloff>:/Icons/WinIcon</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1">
   <item>
    <widget class="QComboBox" name="comboDistribution">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Distribution type</string>
     </property>
     <property name="statusTip">
      <string>Distribution type</string>
     </property>
     <property name="whatsThis">
      <string>Distribution type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QLabel" name="labelUnits">
       <property name="toolTip">
        <string>Units for PSD grid</string>
       </property>
       <property name="statusTip">
        <string>Units for PSD grid</string>
       </property>
       <property name="whatsThis">
        <string>Units for PSD grid</string>
       </property>
       <property name="text">
        <string>Units</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelMax">
       <property name="toolTip">
        <string>Maximum grid limit</string>
       </property>
       <property name="statusTip">
        <string>Maximum grid limit</string>
       </property>
       <property name="whatsThis">
        <string>Maximum grid limit</string>
       </property>
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelEntry">
       <property name="toolTip">
        <string>Grid type</string>
       </property>
       <property name="statusTip">
        <string>Grid type</string>
       </property>
       <property name="whatsThis">
        <string>Grid type</string>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelFun">
       <property name="toolTip">
        <string>Function to define grid</string>
       </property>
       <property name="statusTip">
        <string>Function to define grid</string>
       </property>
       <property name="whatsThis">
        <string>Function to define grid</string>
       </property>
       <property name="text">
        <string>Grid</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="comboUnits">
       <property name="toolTip">
        <string>Units for PSD grid</string>
       </property>
       <property name="statusTip">
        <string>Units for PSD grid</string>
       </property>
       <property name="whatsThis">
        <string>Units for PSD grid</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelMin">
       <property name="toolTip">
        <string>Minimum grid limit</string>
       </property>
       <property name="statusTip">
        <string>Minimum grid limit</string>
       </property>
       <property name="whatsThis">
        <string>Minimum grid limit</string>
       </property>
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineMin">
       <property name="toolTip">
        <string>Minimum grid limit</string>
       </property>
       <property name="statusTip">
        <string>Minimum grid limit</string>
       </property>
       <property name="whatsThis">
        <string>Minimum grid limit</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboFun">
       <property name="toolTip">
        <string>Function to define grid</string>
       </property>
       <property name="statusTip">
        <string>Function to define grid</string>
       </property>
       <property name="whatsThis">
        <string>Function to define grid</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboEntry">
       <property name="toolTip">
        <string>Grid type</string>
       </property>
       <property name="statusTip">
        <string>Grid type</string>
       </property>
       <property name="whatsThis">
        <string>Grid type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelClasses">
       <property name="statusTip">
        <string>Number of classes</string>
       </property>
       <property name="accessibleName">
        <string>Number of classes</string>
       </property>
       <property name="accessibleDescription">
        <string>Number of classes</string>
       </property>
       <property name="text">
        <string>Classes</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinClasses">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Number of classes</string>
       </property>
       <property name="statusTip">
        <string>Number of classes</string>
       </property>
       <property name="whatsThis">
        <string>Number of classes</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineMax">
       <property name="toolTip">
        <string>Maximum grid limit</string>
       </property>
       <property name="statusTip">
        <string>Maximum grid limit</string>
       </property>
       <property name="whatsThis">
        <string>Maximum grid limit</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CQtTable" name="tableGrid">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <column/>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CQtTable</class>
   <extends>QTableWidget</extends>
   <header location="global">QtTable.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboDistribution</tabstop>
  <tabstop>spinClasses</tabstop>
  <tabstop>comboEntry</tabstop>
  <tabstop>comboFun</tabstop>
  <tabstop>lineMin</tabstop>
  <tabstop>lineMax</tabstop>
  <tabstop>comboUnits</tabstop>
  <tabstop>tableGrid</tabstop>
 </tabstops>
 <resources>
  <include location="../../DyssolMainWindow/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
